pkg/virt-operator/resource/generate/rbac/handler.go

```diff
@@ -89,7 +100,6 @@ func newHandlerClusterRole() *rbacv1.ClusterRole {
 					"nodes",
 				},
 				Verbs: []string{
-					"patch",
```

Comment:
Remove the permission means the node patching will fail.

Reply:
Yes, this is intentional.